modulith/build.gradle

```diff
@@ -1,40 +1,32 @@
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '3.1.0' apply false
+    id 'idea'
```

Comment:
we cannot assume that in an opensource system everyone will work with intellij IDEA... build files should be IDE indepenent

Reply:
that's a bit extreme. this plugin does not make the project assume or require anyone to use IDEA, but I removed it, since it is not needed anymore.